lib/translations.json

```diff
@@ -544,6 +548,26 @@
       "key": "metaData/field/twitterPreview",
       "value": "Twitter Vorschau"
     },
+    {
+      "key": "metaData/field/twitterPreview/note",
+      "value": "Sichtbare Textlänge varriert nach Bildschirmgrösse und Ansicht. Auf Mobile wird der Lead meist nicht angezeigt."
+    },
+    {
+      "key": "metaData/field/seoTitle",
+      "value": "SEO Titel (optional)"
+    },
+    {
+      "key": "metaData/field/seoDescription",
+      "value": "SEO Lead (optional)"
+    },
+    {
+      "key": "metaData/field/googlePreview",
+      "value": "Google Vorschau"
+    },
+    {
+      "key": "metaData/field/googlePreview/note",
+      "value": "Der Lead wird allenfalls durch ein Textausschnitt ersetzt sofern es der Suchanfrage mehr entspricht. Sofern nicht explizit definiert erscheinen Twitter oder allgemeine Titel und Leads."
+    },
```

Comment:
This is really not clear. Vorschlag: 

> Google verwendet einen Textausschnitt statt den Lead, falls es mehr zur Suchanfrage passt.

The second part I'd fully drop – we do the same for the Share Tafeln and they see it in the preview don't they?